### PR TITLE
Improved widgets loading priority

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 1]
```diff
@@ -132,6 +132,7 @@
                                 </Grid.ColumnDefinitions>
                                 <FontIcon
                                     Grid.Column="0"
+                                    x:Phase="2"
```
Comment (gave92) [inline Files/UserControls/Widgets/DrivesWidget.xaml:135]: I believe x:Phase makes more sense for big lists that get scrolled, drive list is not big and all the items are always visible at once on screen. Does adding x:Phase make sense here?
[We should really to do it on the GridViewBrowser/GenericFileBrowser template though]

File: Files/UserControls/Widgets/RecentFiles.xaml [hunk 2]
```diff
@@ -102,33 +103,41 @@
...
+                                        x:Phase="2"
                                         Source="{x:Bind FileImg}"
-                                        Stretch="UniformToFill"
-                                        Visibility="{x:Bind FileIconVis}" />
+                                        Stretch="UniformToFill" />
                                 </Grid>
                                 <TextBlock
                                     Grid.Column="1"
                                     VerticalAlignment="Center"
+                                    x:Phase="1"
```
Comment (gave92) [inline Files/UserControls/Widgets/RecentFiles.xaml:132]: Any element without  x:Phase is considered "Phase 0". So you can probably remove the phase from TextBlock and decrease the phase on Images to "1".

File: Files/UserControls/Widgets/DrivesWidget.xaml (hunk 1 above)
Comment (yaichenbaum) [reply Files/UserControls/Widgets/DrivesWidget.xaml:135]: The drives list can get a little long especially when there are a lot of network drives 😃. We should work on this for the GridViewBrowser/GenericFileBrowser templates as well.

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 3]
```diff
@@ -144,17 +145,20 @@
                                         x:Name="ItemLocationName"
                                         Margin="0,0,0,8"
                                         VerticalAlignment="Center"
+                                        x:Phase="1"
```
Comment (gave92) [inline Files/UserControls/Widgets/DrivesWidget.xaml:148]: Could you give this the same treatment as RecentFileList? Remove "Phase 1" and drecrease "Phase 2"?

**Review by gave92 — APPROVED** [review]

**Pull request merged**